anyway/widgets/suburban_widgets/front_to_side_accidents_by_severity.py

```diff
@@ -35,7 +35,7 @@ class FrondToSideAccidentsBySeverity(SubUrbanWidget):
     name: str = "front_to_side_accidents_by_severity"
 
     def __init__(self, request_params: RequestParams):
```

Comment:
Why no call to "calc_widget_digest"?

Reply:
Good catch! I simply forgot to add it to this widget!

Reply:
All calls were moved to the `register` decorator, so there is a single call instead of in each class.